src/poktroll_clients/query_client.py

```diff
@@ -0,0 +1,165 @@
+import asyncio
+from typing import Tuple, Dict
+
+from atomics import INTEGRAL, atomic, INT
+from cffi import FFIError
+
+from poktroll_clients import (
+    go_ref,
+    ffi,
+    libpoktroll_clients,
+    GoManagedMem, BlockQueryClient, Supply, check_err, check_ref,
+)
+from poktroll_clients.proto.poktroll.shared.params_pb2 import Params as SharedParams
+from poktroll_clients.proto.poktroll.application.params_pb2 import Params as ApplicationParams
+from poktroll_clients.proto.poktroll.gateway.params_pb2 import Params as GatewayParams
+from poktroll_clients.proto.poktroll.supplier.params_pb2 import Params as SupplierParams
+from poktroll_clients.proto.poktroll.session.params_pb2 import Params as SessionParams
+from poktroll_clients.proto.poktroll.service.params_pb2 import Params as ServiceParams
+from poktroll_clients.proto.poktroll.proof.params_pb2 import Params as ProofParams
+from poktroll_clients.proto.poktroll.tokenomics.params_pb2 import Params as TokenomicsParams
+from poktroll_clients.protobuf import get_proto_from_go_ref
```

Comment:
Don't think we need these. IIRC, the point was to import them so that they're in the protobuf registry but we're using `importlib` to dynamically load protobuf modules for this effect.